### ISSUE TITLE
TIES with OpenMM 

### ISSUE THREAD

**Issue by Oulfin** [UCL-CCS/TIES#328]
Hi,

Is it possible to use another force field for small molecules like OpenFF? or is it hard to try to implement that?

Is it use Single or Dual topology for the openMM option? Is it possible to choose between single or dual or that is just for Charmm?

Thanks!

**Comment by bieniekmateusz** [comment]
Hi @Oulfin , it would not be difficult to implement and I hope to start working on it soon. 

@adw62 I imagine it is just dual topology atm that mirrors our NAMD setup?

**Comment by adw62** [comment]
Yes I wrote the OpenMM part just to use dual topology and copy NAMD as close as possible. For single topology with OpenMM I'd use [perses](https://github.com/choderalab/perses) I think.

**Comment by bieniekmateusz** [comment]
Please note that perses was also rewritten by Open Force field community as OpenFreeEnergy (https://openfree.energy/). Perses requires OpenEye which was not supported on PowerPC CPUs last year.
